cfme/tests/integration/test_cfme_auth.py

```diff
@@ -420,3 +420,73 @@ def _cleanup():
         for group in retrieved_groups:
             if group.exists:
                 group.delete()
+
+
+@pytest.mark.manual
+@pytest.mark.tier(1)
+@pytest.mark.customer_scenario
+@pytest.mark.meta(coverage=[1710998])
+def test_verify_datastores_list():
+    """
+    Datastore filter should work properly after even delete the datastores
+    Bugzilla:
+        1710998
+    Polarion:
+        assignee: dgaikwad
+        casecomponent: Auth
+        caseimportance: medium
```

Comment:
@dgaikwad since we are adding new coverage shouldn't this be `high`? I thought we weren't adding cases for medium/low.

Reply:
BZ priority is medium so added medium as `casecomponent`. Please let me know what should be our next action